tests/integration-tests/test_runner.py

```diff
@@ -223,6 +239,17 @@ def _set_custom_packages_args(args, pytest_args):
         pytest_args.extend(["--custom-ami", args.custom_ami])
 
 
+def _set_custom_stack_args(args, pytest_args):
+    if args.vpc_cluster:
+        pytest_args.extend(["--vpc-cluster", args.vpc_cluster])
+
+    if args.cluster:
+        pytest_args.extend(["--cluster", args.cluster])
+
+    if args.no_delete:
+        pytest_args.extend(["--no-delete", True])
```

Comment:
this will be `pytest_args.append("--no-delete")` if we go for the flag option

Reply:
changed